ci-operator/jobs/openshift/release/openshift-release-release-4.2-periodics.yaml

```diff
@@ -123,6 +123,90 @@ periodics:
     - name: pull-secret
       secret:
         secretName: ci-pull-credentials
+- agent: kubernetes
+  cron: ""
+  decorate: true
+  interval: 12h
+  labels:
+    job-env: aws
+    job-test: e2e
```

Comment:
These labels don't mean anything for you, you should remove them.  They are used by other jobs that have additional logic to tie them to cluster bot.  So no labels for this job.